mw_url_rewrite.go

```diff
@@ -392,11 +397,13 @@ func checkQueryString(r *http.Request, options map[string]apidef.StringRegexMap,
 		vals, ok := qvals[mv]
 		if ok {
 			for i, v := range vals {
-				b := mr.Check(v)
-				if len(b) > 0 {
-					kn := "trigger-" + strconv.Itoa(triggernum) + "-" + mv + "-" + strconv.Itoa(i)
+				match := mr.FindStringSubmatch(v)
```

Comment:
I've noticed logic like:
```
match := mr.FindStringSubmatch(v)
if len(match) > 0 {
	kn := buildTriggerKey(triggernum, mv, i)
	contextData[kn] = match[0]
	addGroupsToContextData(&contextData, kn, match[1:])
```
repeated in several places, might worth to move it to function if possible